daos/mainnet/dxdao.json

```diff
@@ -28,5 +28,23 @@
     "alias": "GenericSchemeENSRegistryWithFallback",
     "address": "0x9cea0dd05c4344a769b2f4c2f8890eda8a700d64",
     "arcVersion": "0.0.1-rc.36"
+  },
+  {
+    "name": "GenericScheme",
+    "alias": "GenericSchemeOmenTokenRegistry",
+    "address": "0x1f1d9cd773c762ccdd8216ce77a0574105cb4c39",
```

Comment:
the vote param in this Scheme is zero https://etherscan.io/address/0x1f1d9cd773c762ccdd8216ce77a0574105cb4c39#readContract